sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsClient.java

```diff
@@ -454,6 +454,16 @@ public void deleteRelationship(String digitalTwinId, String relationshipId) {
     // Models APIs
     //==================================================================================================================================================
 
+    /**
+     * Creates one or many models.
+     * @param models The list of models to create. Each string corresponds to exactly one model.
+     * @return A {@link PagedIterable} of created models and the http response.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedIterable<ModelData> createModels(List<String> models) {
```

Comment:
Thanks for adding this!

Reply:
I will be changing it back to List ...

Reply:
Good to know. I'll update the e2e tests accordingly